src/DashboardService.Business/ChangeLog/EditChangeLogCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using LT.DigitalOffice.DashboardService.Business.ChangeLog.Interfaces;
+using LT.DigitalOffice.DashboardService.Data.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.ChangeLog;
+using LT.DigitalOffice.Kernel.Responses;
+using Microsoft.AspNetCore.JsonPatch;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Business.ChangeLog;
+
+public class EditChangeLogCommand : IEditChangeLogCommand
+{
+  private readonly IChangeLogRepository _repository;
+
+  public EditChangeLogCommand(IChangeLogRepository repository)
+  {
+    _repository = repository;
+  }
+
+  public Task<OperationResultResponse<bool>> ExecuteAsync(Guid id, JsonPatchDocument<PatchChangeLogRequest> request)
```

Comment:
It is better to change name of PatchChangeLogRequest on EditChangeLogRequest, like get, create...

Reply:
+